local_pathfinding/node_navigate.py

```diff
@@ -141,6 +154,35 @@ def get_desired_heading(self) -> float:
         # TODO: create function to compute the heading from current position to next local waypoint
         return 0.0
 
+    def LPath_callback(self):
+        """Get and publish the local path."""
+
+        waypoints = self.get_LPath()
+
+        local_path = Path(waypoints=waypoints)
+
+        msg = LPathData(local_path=local_path)
```

Comment:
Same thing occurs here, if you feel inclined to change the name of `local_path` to `current_local_path` or something